src/server/jobs/update-videores.js

```diff
@@ -13,40 +12,42 @@ const client = require('../models/db');
 async function main() {
   const logger = await LogServices.createLogger('update-videores');
   const dbclient = await client.connect(process.env.DB_URL);
+  logger.info(`***********************`);
   logger.info(`Connected to database`);
 
   let result = [];
   const srsShortnameMap = [];
   
-  const c = new FtpClient();
 // sftp connectivity
-  //const c = new sftpClient();
-  c.on('ready', function () {
-    c.get(process.env.SECRET_VIDEORES_FILEPATH, function (err, stream) {
-      if (err) throw err;
-      stream.once('close', function () {
-        c.end();
-      });
-      let str = '';
-      stream.on('data', (data) => {
-        str += data.toString();
-      });
-      stream.on('end', async () => {
-        result = UpdateVideoResServices.readStrIntoFields(str, logger);
-      });
-    });
-  });
-  c.once('end', async function () {
+  const sftpClient = new sftpClient();
```

Comment:
@bkoraganti There is a linting issue here, maybe just rename this to c for the client as it was in the previous code?

Reply:
@bkoraganti When I tried to run the script I got the error:

(node:59142) UnhandledPromiseRejectionWarning: ReferenceError: Cannot access 'sftpClient' before initialization
    at main (/Users/rexlorenzo/Projects/lti-media-resources/src/server/jobs/update-videores.js:22:22)